pkg/cli/rack.go

```diff
@@ -426,6 +433,47 @@ func RackUpdate(rack sdk.Interface, c *stdcli.Context) error {
 	return c.OK()
 }
 
+func RackSync(rack sdk.Interface, c *stdcli.Context) error {
+	c.Startf("Synchronizing rack API URL...")
+	c.Writef("\n")
+
+	host, err := currentHost(c)
+	if err != nil {
+		c.Fail(err)
+	}
+	rname := currentRack(c, host)
+
+	if c.String("name") != "" {
+		rname = c.String("name")
+		s, err := ss.NewSession(&aws.Config{})
+		if err != nil {
+			return err
+		}
+
+		cf := cloudformation.New(s)
+
+		o, err := cf.DescribeStacks(&cloudformation.DescribeStacksInput{StackName: aws.String(rname)})
+		if err != nil {
+			return err
+		}
+		for _, s := range o.Stacks {
+			for _, o := range s.Outputs {
+				if *o.OutputKey == "Dashboard" {
+					c.Writef("url=%s\n", *o.OutputValue)
+				}
+			}
+		}
```

Comment:
Suggestion: get the stack from index 0.

```suggestion
		s := range o.Stacks[0]
	        for _, o := range s.Outputs {
		        if *o.OutputKey == "Dashboard" {
			        c.Writef("url=%s\n", *o.OutputValue)
		        }
	        }
```